back-nest/prisma/schema.prisma

```diff
@@ -8,7 +8,7 @@ datasource db {
 }
 
 model Product {
-  id             Int     @id @default(autoincrement())
+  product_id     Int     @id @default(autoincrement())
```

Comment:
Why did you rename it to product_id? Keep an old id and add another column product_id if needed